test/blackbox-test/test-cases/case175.erl

```diff
@@ -0,0 +1,11 @@
+-module(case175).
```

Comment:
It is issue number!
I think `case175.erl` is a bit confusing name. How about `case_univ_pattern.erl` because `_` is called as [universal pattern](http://erlang.org/doc/apps/erts/absform.html#patterns)?

Reply:
I agree.

Reply:
fixed.